### ISSUE TITLE
Feature Suggestion: facilitate substituting special characters

### ISSUE THREAD

**Issue by IsoNovio** [BambusControl/obsidian-substitutions#2]
I have enjoyed the plugin especially its ability to reverse by tapping backspace.

I want to create the following substitution from `::pb` to `%%PAGEBREAK%%<div style=\"page-break-after: always;\"></div>\n`.
The character `\n` cannot be typed in the settings page.
Now I can only do it by directly modifying the configuration `daga.json` and then reloading the plugin.
It would be great to be able to input line breaks directly in the settings page, just like what [text snippets plugin](https://github.com/ArianaKhit/text-snippets-obsidian) does with `$nl$`.

**Comment by BambusControl** [comment]
Thanks for your suggestion.

Yes, special characters are escaped automatically, however I want this plugin to be as simple as possible.
It's meant for simple characters/shortcuts, not advanced snippets/templates like the plugin you mention.
Is there a reason you don't want to use that plugin for your shortcuts?

As it is, I don't see this being looked at in the coming releases of my plugin.

**Comment by IsoNovio** [comment]
Yes, it makes a lot of sense. It is simpler and more secure this way.
